gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Dec 19 23:35:33 CET 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```

Comment:
Z tego co kojarzę to mieliśmy tego już nie ruszać :D

Reply:
-.- nie wiem czemu u mnie to się zmienia.... proponowałem aby wrzucić do out